build/llamafile-convert

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+FILE=$1
+TYPE=${2:-both}
+SCRIPTNAME=$(basename $0)
+
+if [ -z "$FILE" ]; then
+  echo "Usage: $SCRIPTNAME <gguf file or url> [cli|server|both]"
+  exit 1
+fi
+
+# if the file starts with http
+if [[ $FILE == http* ]]; then
+  # download the file
+  # if the filename contains ?download=true, remove it
+  FILE=$(echo $FILE | sed 's/?download=true//g')
+  # get the filename
+  FILENAME=$(echo $FILE | sed 's/.*\///g')
+  echo "Downloading $FILENAME"
+  wget -O $FILENAME $FILE
+  # get the filename
+  FILE=$FILENAME
+fi
+
+# replace .gguf with .llamafile
+LLAMAFILE_NAME=$(echo $FILE | sed 's/.gguf/.llamafile/g')
+# replace .gguf with -server.llamafile
+LLAMAFILE_SERVER_NAME=$(echo $FILE | sed 's/.gguf/-server.llamafile/g')
+LLAMAFILE_PATH=$(which llamafile)
```

Comment:
Please use `command -v` instead of `which` because the prior is a POSIX-compliant shell builtin that's faster.